packages/search-ui/src/__tests__/actions/addFilter.test.js

```diff
@@ -61,25 +61,41 @@ describe("#addFilter", () => {
 
   it("Adds a new filter", () => {
     expect(
-      subject("test", "value", { initialFilters: [{ initial: ["value"] }] })
-        .filters
-    ).toEqual([{ initial: ["value"] }, { test: ["value"] }]);
+      subject("test", "value", {
+        initialFilters: [{ field: "initial", values: ["value"], type: "all" }]
```

Comment:
A lot of tests had to be updated to simply use the new "filter" syntax in state.

Instead of:
```
[
  {[field]: "value"}
]
```

It's now:
```
[
  {
    field: "field",
    value: "value",
    type: "and" // For future and/or/none filters
  }
]

```